content/concepts/basics/03-content-relations.md

```diff
@@ -4,17 +4,68 @@ metaTitle: "sensenet - Content Relations"
+
+The following apply to the behavior of the field:
+
+- **number of references:** a reference field can be configured to hold either one or more references
+- **default reference:** one or more default references can be set via field configuration
+- **types of referable content:** types of referable content can be restricted
+- **location of referable content:** location of referable content can be restricted
+- **set of referable content:** set of referable content can be defined with an optional query
+- **moving/renaming references:** when a reference is moved to another place or it has been renamed, it does not affect the content it is referenced by - the moved/renamed reference is still visible in the feference field
+- **permission handling:** if the current user does not have see permissions to one of the references, that reference is not visible for the user
+- **copying content with references:** when a content with a reference field is copied, the newly created content will hold the same references as the source content. 
+
+(Maybe it's not that important here, could be deleted:
+Copying source content references along with the source content does not affect this behavior, the newly created content will hold references to the originally referenced content and not the newly created copies.)
+
+## Configuration
+The following properties can be set in the field's ``FieldSetting`` configuration:
+
+- **AllowMultiple:** a boolean property defining whether multiple references are allowed or only a single content can be referenced. By default only a single reference is allowed.
+- **AllowedTypes:** allowed content types can be defined by explicitely listing type names in type elements. By default all content types can be referenced.
+- **SelectionRoot:** allowed location of referable content can be defined by listing paths in path elements. By default content can be referenced from under ``/Root``.
+- **DefaultValue:** a default single content reference can be defined with its path. Default multiple references can be defined with a comma separated list. By default the field contains no references.
+
 # Binaries
-https://wiki.sensenet.com/Binary_Field itt is csak az elv a lényeg
+
+Binary field is used for storing binary data. This is the most important field defined on the file content type. After uploading a file, the binary content of your file will be stored in a binary field.
+
+This field can store any kind of binary without length restrictions. This is very useful when you want to store uploaded files in your solution.
+
+> when uploading a ``.docx`` file (for example) to the content repository, it basically creates a new content where the binary is stored in the corresponding field of the new content item.
+
 # Attachments
-itt nagyjából annyi, hogy a két fenti dologgal lehet megoldani végülis
+Following the reference and binary field logic, it's easy to create an attachment for a content by combining these two content relations.
```

Comment:
annyival kiegészítve, hogy eddig 1:1 kapcsolatról volt szó, de ilyen módon lehet 1:több kapcsolatot is kialakítani